### PR TITLE
Translating templates

### LINKED ISSUE #208: Traduzir Repositório
## Comportamento esperado

Read Me, Templates, documentos de configuração todos estão em português. Acredito que deveríamos traduzi-los todos para ingles para internacionalizar o respositório.

## Critérios de Aceitação
Traduzir:
- [x] Read Me
- [x] Template de Issue
- [x] Template de Pr
- [x] Instalação do Docker

## Checklist

- [x] A issue possui Labels.
- [ ] A issue possui prints de tela quando necessário.
- [x] A issue possui nome significativo.

### PR DESCRIPTION
**Pull request by alaxalves**
## Proposed Changes

Fixes #208 
Translated README, CONTRIBUTING and CODE_OF_CONDUCT

## Type of Change

What kind of changes this Pull Request introduces to Falko?
_`x` to what it applies_

- [x] Bugfix
- [ ] New Feature

## Checklist

_`x`to what it applies. If not sure about some topics, don't hesitate to ask. We're here to help!_

- [x] Pull Request's name is self-explanatory and its in English.
- [x] There are no _Rubocop_ issues.
- [x] Travis build has success (tests, code climate and codacy checks).
- [x] Pull Request is linked to an existing issue.
- [ ] Seeds have been created to the new User Stories.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,26 +29,121 @@
...
+
+To contribute with us, the colaborator must _fork_ and send a [pull request](https://github.com/fga-gpp-mds/Falko-2017.2-BackEnd/pulls) with his/her contribution to _devel_ branch.
+The code will be analized by one of the project's owners and, if approved, included to the application's core.
+
+## License
 
 [MIT](https://github.com/fga-gpp-mds/Falko-2017.2-BackEnd/blob/devel/LICENSE)
 
-Copyright (c) 2017 Falko - UnB-FGA (GPP/MDS) Grupo 8 Segundo Semestre de 2017
+Copyright (c) 2018 Falko Organization
```
Comment (cantuariavc) [inline README.md:149]: Here, you have placed 2018 in the year of the copyright and the LICENSE is with the year 2017, @alaxalves.

**Review by cantuariavc — COMMENTED** [review]
Missing the date, looks good.

**Comment by alaxalves** [comment]
@cantuariavc What date you mean?

**Comment by cantuariavc** [comment]
Copyright's date

File: LICENSE [hunk 2]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2017 Falko - UnB-FGA (GPP/MDS) Grupo 8 Segundo Semestre de 2017
+Copyright (c) 2018 Falko Organization
```
Comment (alaxalves) [inline LICENSE:3]: @cantuariavc Fixed the LICENSE date

**Pull request merged**